### PR TITLE
Move to wow.js fork

### PR DESCRIPTION
**Pull request by ntolia**
## Description

wow.js got relicensed. Switch to the MIT fork (the old license)

## Does this PR need a docs update or release note?

- [x] :no_entry: No 

## Type of change

- [x] :bug: Bugfix

## Test Plan

- [x] :muscle: Manual

### CODE REVIEW

File: website/src/components/parts/KeyLoveFAQ.js [hunk 1]
```diff
@@ -5,12 +5,12 @@ export default function KeyLoveFAQ() {
...
   useEffect(() => {
     if (typeof window !== "undefined") {
-      const WOW = require("wowjs");
+      const WOW = require("wow.js");
       const father = require("feather-icons");
       const jarallax = require("jarallax");
       require("tw-elements");
 
-      new WOW.WOW({
+      new WOW({
```
Comment (ntolia) [inline website/src/components/parts/KeyLoveFAQ.js:13]: Not sure why this change was required but it works.

**Review by gmatev — APPROVED** [review]
